src/services/svc-service.ts

```diff
@@ -107,6 +108,14 @@ export class SvcService {
     return this.processDockerExecution(DockerServiceType.STOP, services)
   }
 
+  public restartServices(services: string[]): Promise<ProcessExecutionResult> {
+    this.precheckEnabledServices(services)
+
+    SvcService.debug(`restarting service ${services.join(', ')}`)
+
+    return this.processDockerExecution(DockerServiceType.RESTART, services)
```

Comment:
this should be `executeDockerComposeCommand`

Reply:
done